net-libs/libotr/libotr-4.1.1.ebuild

```diff
@@ -10,13 +10,16 @@ SRC_URI="https://otr.cypherpunks.ca/${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos"
-IUSE=""
 
 RDEPEND="
 	dev-libs/libgcrypt:0=
 	dev-libs/libgpg-error:0="
 DEPEND="${RDEPEND}"
 
+PATCHES=(
+	"${FILESDIR}/${PN}-4.1.1-fix-build-with-libgcrypt-1.10.patch"
+)
+
 src_configure() {
 	econf --disable-static
```

Comment:
This is passed by default with `econf` in [EAPI 8](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-128001r8) ;-)